src/OrbitQt/MainThreadExecutorImplTest.cpp

```diff
@@ -15,8 +16,8 @@ static int argc = 0;
 TEST(MainThreadExecutorImpl, Schedule) {
   QCoreApplication app{argc, nullptr};
 
-  MainThreadExecutorImpl executor{};
-  (void)executor.Schedule([]() { QCoreApplication::instance()->exit(42); });
+  auto executor = MainThreadExecutorImpl::Create();
+  (void)executor->Schedule([]() { QCoreApplication::exit(42); });
```

Comment:
do you need explicit cast to void here? Doesn't it return Future<void>?

Reply:
Ahhhh. Noooo. And yes. From the last code review I reverted the nodiscard in ThreadPool, but not in MainThreadExecutor. Which makes no sense, I guess.

I added a commit which reverts the nodiscard and removes all the unnecessary (void) casts.